scalopus_tracing/src/native/native_tracepoint.cpp

```diff
@@ -30,6 +30,8 @@
 #include <time.h>
 #include <iostream>
 
+#include <nlohmann/json.hpp>
```

Comment:
Hmm, I'm surprised this worked without changes to the `CMakeLists.txt`. I thought I removed [this](https://github.com/iwanders/scalopus/blob/master/scalopus_tracing/CMakeLists.txt#L109) entry to `nlohmann::json`. I don't think it's used anywhere anymore...

Reply:
Removed this dependency from this library.